themes/barbuk/README.md

```diff
@@ -9,6 +9,12 @@ A minimal theme with a clean git prompt
 * Current git info
 * Last command exit code (only shown when the exit code is greater than 0)
 
+## Font
+
+A font with SCM glyph is required to display git remote tool logo.
+You can use a font from https://www.nerdfonts.com/ or patch your own font with the tool
+provided by https://github.com/ryanoasis/nerd-fonts
```

Comment:
A couple of notes:
* Let's use `"SVM glyphs"` (plural) since there are several of them
* The theme supports `svn` and `hg` as well as the various `git` hosts
* The characters are only `default` value's and can be overridden by the user in their `.bash_profile` configuration

With these above notes, I'm thinking we could change the sentence to something like:

```
A font with SCM glyphs is required to display the default tool/host logos.
You can use a font from https://www.nerdfonts.com/ or patch your own font with the tool
provided by https://github.com/ryanoasis/nerd-fonts.
You can also override the default variables if you want to use different glyphs or standard ASCII characters.
```